### PR TITLE
Add Graphql and models for wait time

### PR DESCRIPTION
**Pull request by jyc979**
Modify graphql and schema to add wait time and make helper class + modify dininghall model to incorporate wait time

### CODE REVIEW

File: app/src/main/graphql/com/cornellappdev/android/eatery/eateryQuery.graphql [hunk 1]
```diff
@@ -39,6 +39,13 @@ query AllEateries {
                 }
             }
         }
+        swipeData {
+		    startTime
```
Comment (connorreinhold) [inline app/src/main/graphql/com/cornellappdev/android/eatery/eateryQuery.graphql]: spacing/ this is double indented

File: app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java [hunk 2]
```diff
@@ -162,6 +168,18 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
 
         mId = eatery.id();
 
+        for (AllEateriesQuery.SwipeDatum swipeDatum : eatery.swipeData()) {
```
Comment (connorreinhold) [inline app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java]: Was talking to designers yesterday and I think the plan is to do 30min/hour long increments from 6am to 3am the next day. Is this what the backend supports/does it still return data if there were no swipes in that interval? If not we might want to do some padding of swipedata here to mSwipeDataList by adding intervals with density=0 at the beginning of the day and such

Comment (jyc979) [reply app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java]: Yep, I'm gona ask Conner tmrw about what backend returns for future times that haven't happened yet

Comment (jyc979) [reply app/src/main/java/com/cornellappdev/android/eatery/model/DiningHallModel.java]: talked to backend, so for time intervals that have not happened yet they use historical data, so swipe density ~ wait time high wouldn't be 0. I also confirmed right now(4:30PM) that all future times have some value.

File: app/src/main/graphql/com/cornellappdev/android/eatery/eateryQuery.graphql (hunk 1 above)
Comment (jyc979) [reply app/src/main/graphql/com/cornellappdev/android/eatery/eateryQuery.graphql]: resolved.

**Pull request merged**